common/app/conf/switches/JournalismSwitches.scala

```diff
@@ -62,4 +62,14 @@ trait JournalismSwitches {
     sellByDate = never,
     exposeClientSide = false,
   )
+
+  val BorkWebVitals = Switch(
+    SwitchGroup.Journalism,
+    "bork-web-vitals",
+    "Enables borking (synthetic delay) of web vitals",
+    owners = Seq(Owner.withName("Open Journalism")),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = false,
```

Comment:
is this def false?

Reply:
hmm yeah i suppose that's what i've assumed, i don't know if it's true (about dcr). if dcr still gets it (which it _must_ right?) then great 👍

Reply:
Actually, you were right to be concerned: `.filter(_.exposeClientSide)`

https://github.com/guardian/frontend/blob/2932717e2db3066fc21222d618bb21c7df62049b/common/app/model/dotcomrendering/DotcomRenderingDataModel.scala#L386-L390